superset/commands/report/log_prune.py

```diff
@@ -46,7 +46,7 @@ def run(self) -> None:
                 )
                 try:
                     row_count = ReportScheduleDAO.bulk_delete_logs(
-                        report_schedule, from_date, commit=False
+                        report_schedule, from_date, commit=True
```

Comment:
This is somewhat interesting. With `commit=False` I don't see where this command actually commits. My sense is there should be a `db.session.commit()` on line 60 to ensure we commit once rather than for every report schedule.

BTW I'm working on a PR which will ensure that we handle the "unit of work" correctly.

Reply:
@fisjac you marked this as resolved, yet I don't believe the comment was addressed, i.e., the `run` method should commit once—at line 60 per the previous suggestion—rather than on a per report schedule basis.

Reply:
@john-bodley I had forgotten to push some changes up. This should be covered in the latest push.